src/main.js

```diff
@@ -101,24 +114,37 @@ const listFunctions = async function (srcFolder) {
 }
 
 // List all Netlify Functions files for a specific directory
-const listFunctionsFiles = async function (srcFolder) {
+const listFunctionsFiles = async function (srcFolder, { nodeResolvePaths = [] } = {}) {
```

Comment:
Should `nodeResolvePaths` be added to `listFunctions()` too?

Reply:
Fixed in https://github.com/netlify/zip-it-and-ship-it/pull/302/commits/682cfdd54b1b6b9d657bf98fb6343696940178e6

Reply:
Actually I think `listFunctions` doesn't need it as it doesn't retrieve the source files.

Reply:
Actually I think since `listFunctions` doesn't resolve source files we don't need it.